troubleshooting.md

```diff
@@ -429,3 +429,28 @@ You'll need to either:
 or as root if your user has not enough privileges.
 
 * sudo loginctl enable-linger $UID
+
+### 18) error creating libpod runtime: there might not be enough IDs available in the namespace
```

Comment:
#4614 just beat you to the punch with a separate update.  Please rebase and change this to `### 19)`